Mathlib/Topology/MetricSpace/HausdorffDistance.lean

```diff
@@ -92,6 +92,12 @@ theorem infEdist_iUnion (f : ι → Set α) (x : α) : infEdist x (⋃ i, f i) =
   iInf_iUnion f _
 #align emetric.inf_edist_Union EMetric.infEdist_iUnion
 
+lemma infEdist_biUnion {ι : Type*} (f : ι → Set α) (I : Set ι) (x : α) :
```

Comment:
This is another proposed but not strictly necessary API lemma about `biUnion`s. (See comments at `iUnion_biUnion`.)